bootstrap.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Should we name this `docker-entrypoing.sh`?  I think that's the semi-standard.

Also, to make sure `./env.sh` is successful, shouldn't we do:

```bash
./env.sh && nginx -g "daemon off;"
```